src/main/webapp/app/shared/layouts/profiles/profile.service.ts

```diff
@@ -69,7 +74,7 @@ export class ProfileService {
                 });
         }
 
-        return this.profileInfo;
+        return this.profileInfo.pipe(filter((x) => x !== undefined) as OperatorFunction<ProfileInfo | undefined, ProfileInfo>);
```

Comment:
We should use `!=` here to filter `null` as well: https://www.tektutorialshub.com/typescript/typescript-null-undefined-strict-null-checks/

Reply:
Yes, you are correct. Thank you for the provided article.